app/views/compute_resources_vms/form/vmware/_volume.html.erb

```diff
@@ -1,3 +1,4 @@
+<%= selectable_f f, :storage_pod, vsphere_storage_pods(compute_resource), { :include_blank => _('Deploy VM on selected datastore') }, :class => "span5 without_select2", :label => _("Datastore Cluster"), :label_size => "col-md-3", :onchange => 'vsphereStoragePodSelected(this)' if new_host and vsphere_storage_pods(compute_resource).any? %>
 <%= selectable_f f, :datastore, vsphere_datastores(compute_resource), { }, :class => "span5 without_select2", :label => _("Data store"), :label_size => "col-md-3" %>
```

Comment:
This should be disabled when a storage pod is selected. Any ideas how this can be done as there might be multiple volumes...?